README.md

```diff
@@ -16,6 +16,8 @@ See some of the projects we've built:
 | [nuxt-pinia-tailwind](/nuxt-pinia-tailwind)               | Nuxt Pinia Tailwind CSS          | [https://nuxt-pinia-tailwind.starter.dev](https://nuxt-pinia-tailwind.starter.dev)               |
 | [svelte-kit-scss](/svelte-kit-scss)                       | Svelte SCSS                      | [https://svelte-kit-scss.starter.dev](https://svelte-kit-scss.starter.dev)                       |
 | [vite-apollo-vanilla](/vite-apollo-vanilla)               | Vite Apollo Vanilla              | [https://vite-apollo-vanilla.starter.dev](https://vite-apollo-vanilla.starter.dev)               |
+| [solidstart-tanstackquery-tailwind-modules](/solidstart-tanstackquery-tailwind-modules) | SolidStart TanStack Query Tailwind CSS, CSS Modules       | [https://solidstart-tanstackquery-tailwind-modules.starter.dev](https://solidstart-tanstackquery-tailwind-modules.starter.dev) |
```

Comment:
There is some inconsistency with comma usage on this document, but the first 6 use commas to separate technologies. I'd rather see commas used fully here and update Nuxt, Svelte and Vite  entries to also use commas as technology separators
```suggestion
| [solidstart-tanstackquery-tailwind-modules](/solidstart-tanstackquery-tailwind-modules) | SolidStart, TanStack Query, Tailwind CSS, CSS Modules       | [https://solidstart-tanstackquery-tailwind-modules.starter.dev](https://solidstart-tanstackquery-tailwind-modules.starter.dev) |
```

(also this table uses 3 different ways of listing Tailwind: **Tailwind CSS**, **Tailwind** and **TailwindCSS**)